src/main/resources/listLookup.json

```diff
@@ -248,5 +248,15 @@
         "welshJurisdictions": ["Llys Ynadon"],
         "restrictedProvenances": [],
         "defaultSensitivity": ""
+    },
+    "CIC_DAILY_HEARING_LIST": {
+        "friendlyName": "Criminal Injuries Compensation Daily Hearing List",
+        "welshFriendlyName": "Rhestr Gwrandawiadau Dyddiol y Digolledu am Anafiadau Troseddol",
+        "shortenedFriendlyName": "CIC Daily Hearing List",
+        "url": "",
+        "jurisdictions": ["Tribunal"],
```

Comment:
This will need updating to Criminal Injuries Compensation Tribunal, and same for Welsh jurisdiction

Reply:
Updated